common/src/main/java/com/alibaba/nacos/common/utils/StringUtils.java

```diff
@@ -312,6 +312,20 @@ public static boolean containsIgnoreCase(final CharSequence str, final CharSeque
         return str1.contains(str2);
     }
     
+    /**
+     * Checks if CharSequence contains a search CharSequence.
+     *
+     * @param str       the CharSequence to check, may be null
+     * @param searchStr the CharSequence to find, may be null
+     * @return true if the CharSequence contains the search CharSequence
+     */
+    public static boolean contains(final CharSequence str, final CharSequence searchStr) {
```

Comment:
https://github.com/alibaba/nacos/pull/6265/files

EmbeddedStoragePersistServiceImpl.java should this class also should use this method？

Reply:
Can I modify him in this PR？